corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/ui/animation/Interporators.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.droidkaigi.confsched2020.ui.animation
+
+import android.animation.TimeInterpolator
+import androidx.core.view.animation.PathInterpolatorCompat
+
+/**
+ * Standard easing.
```

Comment:
Thank you!
Please add a link you copied? and Can you add it in README? 🙏 

```
/**
 * @see http://github.com/google/material...
```

Reply:
🙆‍♂

Reply:
@takahirom 
Hi, what about this 26e8069 ?

I copied some files instead of a class.
So, I think comment is more better than KDoc. 🤔